### DISCUSSION TITLE
😉 Submit Achievements for Consideration

### DISCUSSION THREAD

**Discussion by Gary-H9** [community/community#18728] [Other Feature Feedback, Questions, & Ideas]
Can I ask - can we submit our own achievement badges for consideration? 

I'm thinking, for example, that I really like using the [gitmoji](https://github.com/carloscuesta/gitmoji) project and it would be cool for users to be able to gain achievements for using gitmojis in commits etc. 

That's just one example I can think of, off of the top of my head. I'm sure the good people of GitHub can think of others. 

Alternatively it would be interesting to be able to have _organisation specific_ achievements that organisational administrators could create and users within organisations could earn.

**Comment by AndyAn7** [comment]
i agree. please accept this as answer.
